docs/concepts/key-terms/key-terms.mdx

```diff
@@ -1,21 +1,21 @@
 ---
-title: Key Terms
+title: Key Terms & Features
 sidebar_order: 05
-description: "Learn about the key concepts and features of Sentry."
+description: "A few key terms and features to help you get a better understanding of Sentry."
 ---
 
-These key terms and concepts are the fundamentals of Sentry. While we define new terms as you come across them throughout our documentation, having an understanding of the key concepts and features here will make it easier to learn more complex ideas about the product.
+Some of the bellow terms have corresponding features with the same name listed (in upper case) in the [Key Features](#key-features) section.
 
-## Key Concepts
-
-Some of the following key concepts have corresponding features with the same name listed (in upper case) in the [Key Features](#key-features) section.
+## Key Terms
 
 - **alerts** - Alerts let you know about problems with your code in real-time by sending you notifications when certain alert rule conditions are met. There are several types of alerts available with customizable thresholds and integrations.
 
 - **attachments** - Stored additional files, such as config or log files that are related to an error event.
 
 - **data** - Anything you send to Sentry. This includes, events (errors or transactions), attachments, and event metadata.
 
+- **distributed trace** - When a trace includes work in multiple services it's called a [distributed trace](/concepts/key-terms/tracing/distributed-tracing/), because the trace is distributed across those services.
```

Comment:
```suggestion
- **distributed trace** - When a trace includes work in multiple services, it's called a [distributed trace](/concepts/key-terms/tracing/distributed-tracing/), because the trace is distributed across those services.
```